client/state/stats/lists/selectors.js

```diff
@@ -59,7 +59,7 @@ export const getSiteStatsPostStreakData = createSelector(
 
 		if ( streakData && streakData.data ) {
 			Object.keys( streakData.data ).forEach( ( timestamp ) => {
-				const postDay = i18n.moment.unix( timestamp );
+				const postDay = i18n.moment.unix( timestamp ).locale( 'en' );
```

Comment:
Does moment support all of our locales? We can get the current user locale from the current user in the redux tree:

```javascript
import { getCurrentUserLocale } from 'state/current-user/selector';
getCurrentUserLocale( state );
```